recipes-kernel/linux/5.10/patches/xsa-155-qsb-023-xen-netfront-copy-response-out-of-shared-buffer-befo.patch

```diff
@@ -108,14 +110,14 @@ PATCHES
 +		if (unlikely(rx.status < 0 ||
 +			     rx.offset + rx.status > XEN_PAGE_SIZE)) {
  			if (net_ratelimit())
--				dev_warn(dev, "rx->offset: %u, size: %d\n",
--					 rx->offset, rx->status);
+ 				dev_warn(dev, "rx->offset: %u, size: %d\n",
+ 					 rx->offset, rx->status);
```

Comment:
Similiarly this still should remove the `rx->` lines.  I think this patch doesn't compile.